template/public/preload/about.js

```diff
@@ -10,3 +10,9 @@ const contextMenu = require('electron-context-menu');
 contextMenu({
   window: remote.getCurrentWindow(),
 });
+
+window.getVersions = () => [
```

Comment:
You can using `process` directly in React code by adding `window.`. For example: `window.process.versions.electron` so this code can be moved to `src`